pkg/pgclient/client.go

```diff
@@ -180,9 +183,13 @@ func (c *Client) Close() {
 	}
 }
 
+func (c *Client) Ingestor() *ingestor.DBIngestor {
+	return c.ingestor
+}
+
 // Ingest writes the timeseries object into the DB
-func (c *Client) Ingest(tts []prompb.TimeSeries, req *prompb.WriteRequest) (uint64, error) {
-	return c.ingestor.Ingest(tts, req)
+func (c *Client) Ingest(tenant string, tts []prompb.TimeSeries, req *prompb.WriteRequest) (uint64, error) {
```

Comment:
can't say I love this change, but I don't have a good idea of how to avoid. Maybe @antekresic will have an idea here.

Reply:
We can pass there params as single encapsulated struct.

Reply:
Changed this to encapsulated `Request` type.